src/pages/messages/components/NviCandidatesList.tsx

```diff
@@ -32,14 +38,69 @@ export const NviCandidatesList = ({
   helmetTitle,
 }: NviCandidatesListProps) => {
   const { t } = useTranslation();
+  const history = useHistory();
+
+  const searchParams = new URLSearchParams(history.location.search);
+
+  const nviYearParam = searchParams.get(NviCandidatesSearchParam.Year);
+  const nviYearFilter = nviYearParam ? +nviYearParam : nviYearFilterValues[1];
```

Comment:
Har ikke noe imot det, liker gjenbruken, men `nviYearFilterValues[1]` er litt kryptisk framfor å eksponere `currentNviYear`/`thisYear` eller noe fra helper-fila. Men som sagt, bare en tanke som du ikke må gjøre noe med.

Reply:
Sant, er litt kryptisk hva det faktisk er for noe. Senere skal nok hele denne bort da, og i stedet hente alle NVI-perioder som finnes i NVA, og bruke den som har åpen periode som default.
Men tenker jeg legger en `const thisYear = nviYearFilterValues[1]` over komponenten i denne fila i denne omgang for å tydeliggjøre litt mer!